hack/generator/pkg/codegen/pipeline_create_arm_types.go

```diff
@@ -320,24 +319,26 @@ func addArmConversionInterface(
 	kubeDef astmodel.TypeDefinition,
 	armDef astmodel.TypeDefinition,
 	idFactory astmodel.IdentifierFactory,
-	isResource bool) (astmodel.TypeDefinition, error) {
+	isResource bool) (*astmodel.TypeDefinition, error) {
```

Comment:
Same question here? The pointer return type just seems to make the return here more awkward (need to assign to a local to take a reference) and add a dereference at the call site.

Reply:
Reverted.